src/pages/index.js

```diff
@@ -56,6 +63,21 @@ const App = () => {
     download(canvasRef.current);
   };
 
+  const onCopy = (e) => {
+    e.preventDefault();
+    if ("ClipboardItem" in window) {
+      canvasRef.current.toBlob((blob) => {
+        const item = new ClipboardItem({ "image/png": blob });
+        navigator.clipboard.write([item]);
+      });
+      toast("🎉 Copied image!");
+      console.log("copied successfully");
+    } else {
+      toast("Sorry, copying image is not supported on this browser");
```

Comment:
You can fallback to this solution: https://copee.ceriously.com

Reply:
Will do. Thanks

Reply:
@styfle Does `copee` helps with copying images to clipboard? Or just text?

Reply:
@max-programming this answer is relevant: https://stackoverflow.com/a/68241516

Reply:
@sallar Check it now

Reply:
I haven’t tried with an image blob but I suspect it won’t work now that I think about it

Reply:
@max-programming @styfle I think it's not that it's not supported, the issue is that the time delay between the click and the copy call is too long and safari blocks it, the blob needs to be ready when the user clicks the button

Reply:
@styfle @max-programming I fixed the safari issue:
https://github.com/sallar/github-contributions-chart/pull/131/files#diff-64525bbcfc9ec7f8181ea4db7b0d1692090358183eb03af8f1f2711116903537